### ISSUE TITLE
Condemn not being suggested for Fury at 100-80% range.

### ISSUE THREAD

**Issue by gbencivengo** [Hekili/hekili#444]
Hi!

In my testing with Fury Warrior on SL Beta, I've noticed that Condemn is not being properly recognized by the suggested actions. 

1. When I'm on a training dummy that is at 100% health and I charge and ran through the suggested actions, Condemn is never once suggested. I at first thought this to be an issue with the addon just never properly recognizing Condemn, so I went out into the open world to find some mobs to beat on to see if it would suggest Condemn at a more traditional execute range, <= 35% (I'm running Massacre). 

2. I went into the open world and charged and ran through the suggested actions on mobs. After doing that about a couple of times, I noticed that Hekili is still suggesting that I use "Execute", only at sub-35%.

I looked at /classes/WarriorFury.lua - and I noticed that Condemn is no where to be found in there. Although Condemn exists in /classes/WarriorArms.lua.

To recap - I don't think Condemn, specifically at the 100-80 percent range is being properly added/suggested in the action priority list.

Thanks!

**Comment by Hekili** [comment]
Follow the instructions and submit a proper ticket.

Condemn works perfectly fine for Fury, despite being housed in the Arms file.

**Comment by Hekili** [comment]
(There is a separate issue with Covenant ability detection that is impacting Condemn, but will be resolved in the next build.)
